app/scripts/clear-docker.sh

```diff
@@ -1,3 +1,5 @@
 docker stop $(docker ps -aq)
 docker rm $(docker ps -aq)
 docker network prune --force
+
+docker ps
```

Comment:
Hm, I put this in to try and diagnose a bit more about the weird db containers that still seem to be getting in the way on some builds. This script *should* be stopping running containers. But then the next time it ran fine.....

Reply:
try `docker rm -f`